config/database.php

```diff
@@ -74,7 +74,7 @@
             'charset' => 'utf8',
             'prefix' => '',
             'prefix_indexes' => true,
-            'schema' => 'public',
```

Comment:
Hvad vil du ændre med dette? Det er en config variable der angiver hvilken database schema vi skal anvende til at få tables fra... Eller vil du anvende det til noget andet?

Reply:
Det er din ven som har ændret på det. Måske bør vi overveje at droppe det der autocorrection af koden. Jeg ved godt at du er forelsket i den, men den er en bitch for min vedkommende, for man kan ikke rigtigt se hvem som har lavet denne rettelse.

Reply:
Næææ. Min ven laver aldrig breaking changes. 
Den fikser KUN stylemæssigt ændringer som ikke kan risikere nogle skader på koden.

Ændringen kommer fra din allerførste commit 7e7ac355c51ac2daa55711f7d0cd8200a45cfcac - så det er ikke noget automatiseret ting der forstyrrer dette...